src/components/FeatureViews/Courses/Announcements.js

```diff
@@ -56,17 +64,17 @@ const AnnouncementCard = ({
   handleEdit,
   handleDelete,
 }) => {
+  const decode = EditorState.createWithContent(convertFromRaw(JSON.parse(body)))
```

Comment:
Why do you have to `JSON.parse` `body`? Is it not possible to pass in the JSON directly?

Reply:
When the body is loaded from the database, it's loaded as a string so I'd have to parse it back to an object.